app/controllers/sessions_controller.rb

```diff
@@ -1,3 +1,21 @@
 class SessionsController < ApplicationController
-  def new; end
+  def new;end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user && user.authenticate(params[:session][:password])
```

Comment:
You can try using [Safe Navigation Operator ](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)
```ruby
user&.authenticate params[:session][:password]
```